### PR TITLE
Build.sbt cleanup and fix publishMavenStyle

### PR DESCRIPTION
**Pull request by ericdotdata**
…scala_2.12/sbt_1.0/0.1.0-SNAPSHOT/poms/sbt-config.pom pathing

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+scalaVersion := "2.12.12"
+
 homepage := Some(url("https://github.com/ramencloud/sbt-config"))
 licenses := Seq("Apache 2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
 
-name := "sbt-config"
-
 val baseVersion = "0.1"
 
-version in ThisBuild := {
```
[the remark this answers is not in the document]
Comment (ericdotdata) [reply build.sbt:11]: just consistency with other settings, we can move them all to `in ThisBuild` or remove it from all

File: build.sbt [hunk 2]
```diff
@@ -26,7 +26,7 @@ version in ThisBuild := {
   }
 }
 
-publishMavenStyle := true
+publishMavenStyle := false
```
Comment (ericdotdata) [inline build.sbt:29]: I broke this in master, sbt plugins should NOT be publishMavenStyle

**Review by dot-vlad — APPROVED** [review]
I am still not sure about `in ThisBuild`, and what it affects

**Comment by ericdotdata** [comment]
I had to look it up, its a way to scope incase you have multiple sub projects

**Comment by ericdotdata** [comment]
> I had to look it up, its a way to scope incase you have multiple sub projects

Clarifying this: https://www.scala-sbt.org/1.x/docs/Scopes.html#Build-level+settings

**Pull request merged**